### PR TITLE
Add pod stats

### LINKED ISSUE #4: Add pod stats
Add a way to display the number of contributions between fellows. Make a little leaderboard or similar to motivate people to add new issues!

### PR DESCRIPTION
**Pull request by ttn-ttn**
## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [x] I've explained what the Pull Request is adding.
- [x] I've explained why this is important.

This PR fixes #4  by adding a statistics page that displays:
- Number of commits per pod project
- Number of merged pull requests per pod project
- Total number of commits by the pod
- Number of commits per fellow
- Number of merged pull requests per fellow

The statistics are fetched by a plugin that calls twice the Github API. Probably no API key is needed, Github allows a good number of unauthenticated calls per day.

As this PR makes use of the navbar through a template as in PR #22 , it's better to wait until that PR is merged (or rejected).

More importantly, it requires one last update before being ready. That update will be possible when PR #19 is merged.

### CODE REVIEW

File: _plugins/pod_stats.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  require 'json'
+  class StatsGenerator < Jekyll::Generator
+    safe true
+
+    def generate(site)
+      # list of MLH repos the pod is working on
+      projects = site.data['projects']
+      fellows = site.data['fellows'] # list of pod fellows
+      # TODO: update 'name' with the actual key (e.g. 'github')
+      # as soon as issue 13 gets solved
```
Comment (ttn-ttn) [inline _plugins/pod_stats.rb]: Currently waiting for PR #19 to be merged

**Comment by ttn-ttn** [comment]
@akshgpt7 currently merging PR #41 is the only blocker before this can be merged.

**Comment by ttn-ttn** [comment]
@akshgpt7 this is finally ready for review!

**Comment by akshgpt7** [comment]
hey @ttn-ttn sorry, last few merge conflicts to fix, I promise I'll merge this first before any PR now so other conflict don't arise :p

**Comment by akshgpt7** [comment]
I've had a look at the changes, they look super cool!
It's ready to be merged with the conflicts resolved

**Comment by ttn-ttn** [comment]
@akshgpt7 there seems to be a bug still, now that I've added the weather app project the statistics aren't correct. I will fix that tomorrow and mention you in this issue as soon as it's done, you can merge other PRs on the meantime, there shouldn't be any serious merge conflict with the current PRs

**Comment by ttn-ttn** [comment]
@akshgpt7 fixed, it was actually a very minor thing. Now it should be ready for merging :)

**Comment by akshgpt7** [comment]
Perfect! Thank you so much for working on this 💪

**Pull request merged**